cypress/support/pages/homeCatalogue.pageObject.js

```diff
@@ -17,6 +17,44 @@ class HomeAndCataloguePageObject extends PageObject {
     cy.contains('.hrefch', product)
       .click();
   }
+
+  addToCart() {
+    cy.get('[onclick="addToCart(9)"]').click();
+  }
+
+  assertInCart() {
+    cy.findById('tbodyid').should('contain', 'Sony vaio i7');
```

Comment:
The assertion in 'assertInCart' is hardcoded to check for 'Sony vaio i7'. This might not be flexible if you want to assert different products. Consider making this method more dynamic by passing the product name as a parameter.